Kat

km1036 opened the pull request.

km1036 committed:
Update README.md

km1036 committed:
Adding files for json/html preparation via upload

Cleaned 'CO2_emissions_1990_2018.csv' and renamed 'clean_for_json.csv' to prep for conversion to json file ('data.json').

km1036 committed:
Update README.md

km1036 committed:
Add files via upload

km1036 committed:
Add files via upload

km1036 committed:
Delete Mapping_World directory

km1036 committed:
Add files via upload

camillecoding commented:
did you want to overwrite your readMe with the one on the main branch? I know you had info specific to your contributions in there so I don't want you to lose it (unless you don't mind)

km1036 committed:
testing

km1036 committed:
Add files via upload

km1036 committed:
Delete geo_data.json

km1036 committed:
Uploading cleaned and completed CO2 Emissions Data 

Used for plots, graphs, and statistical analyses

km1036 committed:
Map with time slider using Plotly in R 

Global CO2 Emissions since 1750. By clicking "Play," once can view the changes in the CO2 emissions reported over time throughout the world.

km1036 committed:
Update README.md

km1036 committed:
Update README.md

km1036 committed:
HTML/JS interactive map (work in progress)

km1036 committed:
Update README.md
